srcpkgs/ta-lib-devel/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'ta-lib-devel'
+pkgname=ta-lib-devel
```

Comment:
Should be split to ta-lib and ta-lib-devel, like eg. libarchive. Then -devel goes to makedepends and lib to depends.